### ISSUE TITLE
Unified Vocab

### ISSUE THREAD

**Issue by PeterTheHe** [PeterTheHe/JGT#3]
What if the "Consult the DVL" page were just one page with language options? Could just make tables out of whatever data it gets back.

**Comment by DylanHiller** [comment]
Yeah sounds good, but I suppose this is just rather linking it to one database. Perhaps make it so that you can still pull up a list of the words?

**Comment by PeterTheHe** [comment]
Nah all the requests are going to the GAS which takes in language too. We have 2 separate things accessed with a language parameter ("greek" or "latin")
